components/PoliticianWidget/PoliticianWidget.tsx

```diff
@@ -185,7 +189,9 @@ export function PoliticianWidget({
         <>
           <div className={styles.divider} />
           <section className={styles.endorsementSection}>
-            <h4>Endorsements</h4>
+            <h4>
+              Endorsements <FaInfoCircle color="var(--grey-dark)" />
+            </h4>
```

Comment:
This section will have the info circle but it won't do anything.  We should refactor this so that this section is always rendered, and the endorsements / NONE are rendered conditionally